examples/models/spatial/plot_shell2.py

```diff
@@ -0,0 +1,77 @@
+r"""
+.. _shell2-spatial-model:
+
+Shell2 Spatial Model
+===================
+
+This is a spatial model parametrizing a projected radiating shell.
+
+The shell spatial model is defined by the following equations:
+
+.. math::
+    \phi(lon, lat) = \frac{3}{2 \pi (r_{out}^3 - r_{in}^3)} \cdot
+            \begin{cases}
+                \sqrt{r_{out}^2 - \theta^2} - \sqrt{r_{in}^2 - \theta^2} &
+                             \text{for } \theta \lt r_{in} \\
+                \sqrt{r_{out}^2 - \theta^2} &
+                             \text{for } r_{in} \leq \theta \lt r_{out} \\
+                0 & \text{for } \theta > r_{out}
+            \end{cases}
+
+where :math:`\theta` is the sky separation, :math:`r_{\text{out}}` is the outer radius
+and  :math:`r_{\text{in}}` is the inner radius.
```

Comment:
One last comment: could you at the definition of `R_in` with respect to `eta` here?